src/ray/gcs/tables.h

```diff
@@ -377,6 +377,26 @@ class ActorTable : public Log<ActorID, ActorTableData> {
     pubsub_channel_ = TablePubsub::ACTOR;
     prefix_ = TablePrefix::ACTOR;
   }
+
+  /// Append data to actor table at the given location.
```

Comment:
The `ActorTable` is now highly structured, so let's make sure to document the expected order and type of log entries here.

Reply:
I'm not sure what you meant by `expected order and type of log entries`.